src/main/java/prefs/Preferences.java

```diff
@@ -104,29 +196,51 @@ public void setPanelInfo(List<PanelInfo> panelInfo) {
 
     public void addBoard(String name, List<PanelInfo> panels) {
         assert name != null && panels != null;
-        global.addBoard(name, panels);
+        sessionConfig.addBoard(name, panels);
     }
```

Comment:
In future, this should take a `BoardInfo` object. Not related to this PR though.